rusk/Cargo.toml

```diff
@@ -38,7 +38,7 @@ dusk-bls12_381-sign = { version = "0.1.0-rc", features = ["canon"] }
 dusk-jubjub = { version = "0.10", features = ["canon"] }
 dusk-pki = "0.9.0-rc"
 dusk-bytes = "0.1"
-dusk-wallet-core = "0.10.0-rc"
+dusk-wallet-core = { git = "https://github.com/dusk-network/wallet-core", branch = "stake-change-46" }
```

Comment:
Again, this feels a little odd, is this done on purpose?

Reply:
[Ditto](https://github.com/dusk-network/rusk/pull/578#discussion_r808905090)